rke/structure_rke_cluster_ingress.go

```diff
@@ -29,6 +29,8 @@ func flattenRKEClusterIngress(in rancher.IngressConfig) []interface{} {
 		obj["provider"] = in.Provider
 	}
 
+	obj["default_backend"] = *in.DefaultBackend
```

Comment:
As `in.DefaultBackend` is a pointer, this is not safe. It has to be checked that is not nil before assign it

```
-	obj["default_backend"] = *in.DefaultBackend
+	if in.DefaultBackend != nil {
+		obj["default_backend"] = *in.DefaultBackend
+	}
```